examples/kind/README.md

```diff
@@ -0,0 +1,72 @@
+# KIND Demo
+
+[kind](https://github.com/kubernetes-sigs/kind)(Kubernetes in Docker) is a tool for running local Kubernetes clusters using Docker container “nodes”. Azure CSI Driver will work in kind using Service Principal.
+
+## Prerequisite
+
+- Follow [kind installation instruction](https://github.com/kubernetes-sigs/kind#installation-and-usage) to setup kind in your machine
+
+> Windows 10 users could you wsl 2 to install kind and run this sample. Integrate docker for windows with wsl 2 by following the [instructions](https://docs.docker.com/docker-for-windows/wsl-tech-preview/).
+
+## Setup
+
+- Create a Azure AD App to create Service Principal and give it "GET" permission for secrets in keyvault. Follow the steps in [keyvault docs](https://docs.microsoft.com/en-us/azure/key-vault/general/group-permissions-for-apps#applications). Keep `ClientID` and `ClientSecret` of the AD app handy.
+
+- Update `nginx-pod-secrets-store-inline-volume.yaml` to provide keyvault name and the keyvault resources to fetch.
+
+```
+keyvaultName: '' # the name of the KeyVault
+objects: |
+    array:
+        - |
+        objectName: secret1
+        objectType: secret        # object types: secret, key or cert
+        objectVersion: ""         # [OPTIONAL] object versions, default to latest if empty
+        - |
+        objectName: key1
+        objectType: key
+        objectVersion: ""
+resourceGroup: '' # [REQUIRED for version < 0.0.4] the resource group of the KeyVault
+subscriptionId: '' # [REQUIRED for version < 0.0.4] the subscription ID of the KeyVault
+tenantId: '' # the tenant ID of the KeyVault
+```
```

Comment:
these values have incorrectly spaced

Reply:
VSCode put 4 spaces instead of 2. Fixed it. Was there any other issue here?